### DISCUSSION TITLE
Getting to know the users

### DISCUSSION THREAD

**Discussion by Battleman** [Battleman/zoomdl#65] [General]
Hi there! I've been developing this project since about the start of the pandemic (March 2020), to fulfill a need I had. Now this project has (a bit) exploded, and is by far my most popular one. So I'm getting curious as to who is using it. As there is absolutely 0 tracking, I have no idea of the usage. Please let me know about your relation with ZoomDL!

For example, how did you hear of ZoomD? How often do you use it? Why do you prefer it to alternatives? Is something bugging you off?

I'd be delighted to hear from as many people as possible. Possibly to advertise it more to some forums/subreddit, or focus my attention on a core feature unique to my project.

**Comment by mchobbylong** [comment]
Thanks for this great tool!

Using it just because my university offers lecture recordings in limited time, and I personally want to save them locally. This repo was found by simply searching in GitHub.
